core/src/main/java/hudson/PluginManager.java

```diff
@@ -2240,7 +2240,7 @@ private PluginUpdateInfo(String pluginName, String message) {
     @Restricted(NoExternalUse.class)
     public Object getTarget() {
         if (!SKIP_PERMISSION_CHECK) {
-            Jenkins.get().checkPermission(Jenkins.ADMINISTER);
+            Jenkins.get().checkPermission(Jenkins.SYSTEM_READ);
```

Comment:
Use the solution in the large Overall/Manage PR instead, please.

Reply:
Hmm? What are you asking for here?

Reply:
done in https://github.com/jenkinsci/jenkins/pull/4520/commits/1a15e159e0e630622679650895f9b0d6b009dc80